IgrPhillipe/go-go-vaccines
# Feature/multiple background images

What I done:
- Separate the background images and the way they move to the side

How to test:
- Open the game with "python3 Game.py" 
- See if the background is passing with different speeds
